bcipy/signal/model/base_model.py

```diff
@@ -17,8 +17,17 @@ class SignalModelMetadata(NamedTuple):
     transform: Composition  # data preprocessing steps
     evidence_type: str = None  # optional; type of evidence produced
     auc: float = None  # optional; area under the curve
+    acc: float = None  # optional; accuracy
```

Comment:
acc: float = 0.0 ?

Reply:
Knowing when this metadata was set instead of assuming 0% acc is helpful, I think.